src/Pos/Block/Logic/Internal.hs

```diff
@@ -10,8 +10,10 @@ module Pos.Block.Logic.Internal
          -- * Constraints
          MonadBlockBase
        , MonadBlockVerify
+       , MonadBlockApplyDB
        , MonadBlockApply
 
+       , applyBlocksDbUnsafe
```

Comment:
I totally don't like new names in this module. They are absolutely confusing. I would export `applyBlocksUnsafe` which doesn't do normalization and would define `normalizeBlocks` in `VAR` and use it in `verifyAndApplyBlocks` in the end. It should also resolve CSL-544.